FrederickMunro/wiley-team-three
# Implemented Login/Signup pages, connected them to API, implemented cookies to manage logged in users

Description:
This PR tackles many problems.
1. Implementation of UI skeleton for login and signup pages.
2. Connection of this skeleton to backend api calls.
3. Implementation of cookies for tracking logged in users.
4. Addition of @CrossOrigin to controllers to remove CORS errors.

Testing:
1. git checkout frontend
2. git pull
3. run backend
4. Navigate to /frontend directory
5. npm i
6. npm run dev
7. You should be at landing page, this page is only responsive until 1999px of width. So if it looks horrible shrink the page.
8. Click sign up, should be redirected to signup page.
9. Create a new account.
10. If successful will redirect you to landing page, the sign up button should be gone and a log out button should appear in the menu.
12. Press the logout button.
13. Sign up button should reappear.
14. Press sign up, then log in button at the top right.
15. Login in using same username/password as before.
16. Should be redirected to landing page with signup button gone.

If all this is smooth, you can approve. Let me know if you have any ideas/suggestions for improvements!
